build-analyze.ps1

```diff
@@ -0,0 +1,42 @@
+$ErrorActionPreference = 'Stop'
+
+$SONAR_SERVER_URL = "https://sonarcloud.io"
+#$SONAR_TOKEN = # Access token coming from SonarCloud projet creation page. In this example, it is defined in the environement through a Github secret.
+$SONAR_SCANNER_VERSION = "4.6.1.2450" # Find the latest version in the "Windows" link on this page:
+                                      # https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/
+$BUILD_WRAPPER_OUT_DIR = "build_wrapper_output_directory" # Directory where build-wrapper output will be placed
+
+mkdir $HOME/.sonar
+
+# Download build-wrapper
+$path = "$HOME/.sonar/build-wrapper-win-x86.zip"
+rm build-wrapper-win-x86 -Recurse -Force -ErrorAction SilentlyContinue
+rm $path -Force -ErrorAction SilentlyContinue
```

Comment:
These and l.23, l.24 seem to be there just for the script reentrancy, I think they have no effect for a script that runs in a freshly spawned container. I propose to remove them to simplify the example and not scare users